src/components/cookieBanner/cookieBanner.js

```diff
@@ -57,11 +57,11 @@ export function getConsentCookie() {
   const decodedCookie = decodeURIComponent(window.document.cookie);
   const ca = decodedCookie.split(';');
   for (var i = 0; i < ca.length; i++) {
-    const c = ca[i];
-    while (c.charAt(0) == ' ') {
+    let c = ca[i];
+    while (c.charAt(0) === ' ') {
       c = c.substring(1);
     }
-    if (c.indexOf(name) == 0) {
+    if (c.indexOf(name) === 0) {
```

Comment:
This could also be simplified a bit with `c.startsWith(name)`, but I guess it's totally ok with `indexOf` : )
And nice catch on the `==`!